shell/client/shell.html

```diff
@@ -1354,8 +1361,12 @@ <h3 id="app-stats">Apps</h3>
     {{else}}
       <div class="feature-key-button-row">
         <button class="button-primary feature-key-upload-button">Upload new feature key...</button>
+        <button class="button-secondary feature-key-delete-button">Delete feature key</button>
```

Comment:
I question the utility of this.  I don't think anyone who isn't us will actually want this, and it's a bit of a footgun to leave in the product.